bluepymm/prepare_combos/parse_files.py

```diff
@@ -174,9 +183,9 @@ def read_records():
             emodel = etype_map['mm_recipe']
             for layer in etype_map['layer']:
                 for fullmtype in fullmtypes:
-                    if mtype_regex.match(fullmtype):
+                    if fullmatch(mtype_regex, fullmtype):
```

Comment:
why not just make sure the regex ends in '$'?

Reply:
that's another option, but it requires e.g. recompiling the regex

Reply:
And one has to be careful there is no $ yet

Reply:
yeah, I mean before the regex gets compiled: where ever they are added. `process_megate_config.py::join_regex` already does something similar